packages/mason/test/src/render_test.dart

```diff
@@ -62,8 +62,7 @@ void main() {
 
       test('constantCase outputs correct string', () {
         const greeting = 'hello world';
-        const input =
-            'Greeting: {{#constantCase}}{{greeting}}{{/constantCase}}!';
+        const input = 'Greeting: {{#constantCase}}{{greeting}}{{/constantCase}}!';
```

Comment:
Can we revert these changes? The format check is failing because these lines exceed 80 chars.

Reply:
Whoops - had different setting on my env. Fixed!

Reply:
Np!